Src/VimCore/Interpreter_Parser.fs

```diff
@@ -1299,12 +1299,8 @@ type Parser
                     _tokenizer.MoveToMark mark
                     None
 
-            elif _tokenizer.CurrentChar = '+' then
-                _tokenizer.MoveNextToken()
-                x.ParseNumber() |> Option.map LineSpecifier.AdjustmentOnCurrent
-            elif _tokenizer.CurrentChar = '-' then
-                _tokenizer.MoveNextToken()
-                x.ParseNumber() |> Option.map (fun number -> LineSpecifier.AdjustmentOnCurrent -number)
+            elif _tokenizer.CurrentChar = '+' || _tokenizer.CurrentChar = '-' then
+                Some LineSpecifier.CurrentLine

```

Comment:
Ah nice simplification. Does look like I was doing the work for the adjustment case twice here. Had to open it up locally to see where this was already being accounted for.